src/easylink/pipeline_schema_constants/__init__.py

```diff
@@ -1,11 +1,12 @@
-from easylink.pipeline_schema_constants import development, tests
+from easylink.pipeline_schema_constants import development, testing
 
 ALLOWED_SCHEMA_PARAMS = {
     "development": development.SCHEMA_PARAMS,
 }
 
 TESTING_SCHEMA_PARAMS = {
-    "integration": tests.SINGLE_STEP_SCHEMA_PARAMS,
-    "combined_bad_topology": tests.BAD_COMBINED_TOPOLOGY_SCHEMA_PARAMS,
-    "combined_bad_implementation_names": tests.BAD_COMBINED_TOPOLOGY_SCHEMA_PARAMS,
+    "integration": testing.SINGLE_STEP_SCHEMA_PARAMS,
+    "combined_bad_topology": testing.BAD_COMBINED_TOPOLOGY_SCHEMA_PARAMS,
+    "combined_bad_implementation_names": testing.BAD_COMBINED_TOPOLOGY_SCHEMA_PARAMS,
```

Comment:
Note for others (took me a second): the only change here is `tests` -> `testing`